### DISCUSSION TITLE
Dataline fails to start after installing via homebrew on Mac OSX Sequoia 15.0.1

### DISCUSSION THREAD

**Discussion by ashaikh** [RamiAwar/dataline#351] [Q&A] [answered]
Dataline fails to start after installing via homebrew on Mac OSX Sequoia 15.0.1 on a Macbook Pro M3 Max with 64 gb ram. 
Dataline version 1.0..14

Command: dataline

Output: 
pydantic/_internal/_fields.py:200: UserWarning: Field name "api_key" in "ShouldGenerateChartCall" shadows an attribute in parent "OpenAIExtractor[ShouldGenerateChart]"

pydantic/_internal/_fields.py:200: UserWarning: Field name "base_url" in "ShouldGenerateChartCall" shadows an attribute in parent "OpenAIExtractor[ShouldGenerateChart]"

pydantic/_internal/_fields.py:200: UserWarning: Field name "api_key" in "GenerateChartCall" shadows an attribute in parent "OpenAIExtractor[GeneratedChart]"

pydantic/_internal/_fields.py:200: UserWarning: Field name "base_url" in "GenerateChartCall" shadows an attribute in parent "OpenAIExtractor[GeneratedChart]"

pydantic/_internal/_fields.py:200: UserWarning: Field name "api_key" in "ConversationTitleGenerator" shadows an attribute in parent "OpenAICall"

pydantic/_internal/_fields.py:200: UserWarning: Field name "base_url" in "ConversationTitleGenerator" shadows an attribute in parent "OpenAICall"

INFO:     Started server process [37096]

INFO:     Waiting for application startup.

INFO:alembic.runtime.migration:Context impl SQLiteImpl.

INFO:alembic.runtime.migration:Will assume non-transactional DDL.

ERROR:    Traceback (most recent call last):

  File "starlette/routing.py", line 677, in lifespan

  File "contextlib.py", line 204, in __aenter__

  File "dataline/main.py", line 61, in lifespan

AttributeError: 'NoneType' object has no attribute 'id'

 

ERROR:    Application startup failed. Exiting.

**Comment by RamiAwar** [comment, marked as answer]
Thanks for reporting this - just released a new version fixing this! Let me know if the problem persists after upgrading.
